src/actions/arcActions.ts

```diff
@@ -508,6 +520,8 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       const proposal = {
         beneficiary,
         boostedTime: 0,
+        boostedVotePeriodLimit: votingMachineParams[2],
```

Comment:
need to cast to Number right?

Reply:
updated: https://github.com/daostack/alchemy/pull/38/commits/0f0f22d876c6d80100aa110ec022dd755d6c255e